requirements.txt

```diff
@@ -1,7 +1,7 @@
 cogment[generate]==2.7.0
 grpcio==1.43.0
 hydra-core==1.3.2
-mlflow==1.26.1
+mlflow~=1.30.0
```

Comment:
Needed to bump to use the mlflow client environment variables (ex: `MLFLOW_HTTP_REQUEST_MAX_RETRIES`)